.github/workflows/build_with_nix.yml

```diff
@@ -0,0 +1,20 @@
+name: build with nix
+on:
+  schedule:
```

Comment:
Can you give some details why we would not want to run this on PRs? In case of changes we would like to know that this fails?

Reply:
Ah, the moment we don't have a Cachix subscription, so we sint have a central store to place things that we have built before. This includes tket - so when this CI runs, it will need to build tket first.

Once we have a caching subscription, the latest tket build will already be available, so building and testing pytket-qir will just be a case of bundling the python package and running checks on it. Then, if we store the result in cachix, anyone can load up a pre-built environment with tket, pytket and pytket-qir very quickly.

Reply:
Thank you for all the details!